content/en/logs/log_collection/nodejs.md

```diff
@@ -128,6 +128,54 @@ logs:
     sourcecategory: sourcecode
 ```
 
+## Agentless Logging
+
+It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
+
+{{< tabs >}}
+{{% tab "Winston 3.0" %}}
+
+You can use [winston HTTP transport][1] to send your logs directly through the [datadog API][3].
+In your bootstrap file or somewhere in your code, declare the logger as follow:
+
+```js
+
+const { createLogger, format, transports } = require('winston');
+
+const httpTransportOptions = {
+  host: 'http-intake.logs.datadoghq.com',
+  path: '/v1/input/<APIKEY>?ddsource=nodejs&service=<APPLICATION_NAME>',
+  ssl: true
+};
+
+const logger = createLogger({
+  level: 'info',
+  exitOnError: false,
+  format: format.json(),
+  transports: [
+    new transports.Http(httpTransportOptions),
+  ],
+});
+
+module.exports = logger;
+
+// Example logs
+logger.log('info', 'Hello simple log!');
+logger.info('Hello log with metas',{color: 'blue' });
+```
+
+__Note:__ To send logs to Datadog EU site, set the host property to `http-intake.logs.datadoghq.eu`
+
+If you are using US site, you can also check the community supported [Datadog Transport][2].
```

Comment:
If we recommend this transport, should we try to ask for its ownership?

Reply:
Asking for the ownership looks like a good idea indeed if you think that the level of quality is good and a Datadog team accept to maintain it. IMO we should discuss this as a separated thread.